src/language.js

```diff
@@ -19,8 +19,8 @@ const error = {
   503: 'Server error: please contact support@datamermaid.org',
   apiDataSync: 'The app was not able to sync data with the API. Please try again.',
   appNotAuthenticatedOrReady: 'Initialization error. Try reloading or reauthenticating.',
-  collectRecordChoicesUnavailable:
-    'Options data for creating a sample unit is currently unavailable. Please try again',
+  collectRecordSupportingDataUnavailable:
+    'Supporting data for creating a sample unit is currently unavailable. Please try again',
```

Comment:
I dont know how this weird language/logic evolved, but we/I copied and pasted its use for all the collect recorrds. oof

Reply:
makes more sense!